app/services/moves/finder.rb

```diff
@@ -2,11 +2,9 @@
 
 module Moves
   class Finder
-    attr_reader :filter_params, :ability
+    attr_reader :filter_params, :ability, :db_includes
 
-    MOVE_INCLUDES = [:allocation, :supplier, :court_hearings, :prison_transfer_reason, :original_move, profile: [documents: { file_attachment: :blob }, person_escort_record: [:framework, :framework_responses, framework_flags: :framework_question]], person: %i[gender ethnicity], from_location: %i[supplier_locations], to_location: %i[supplier_locations]].freeze
-
-    def initialize(filter_params, ability, order_params)
+    def initialize(filter_params, ability, order_params, db_includes)
```

Comment:
I think now would be a good time to introduce named parameters for this class - three params was just about tolerable but four would be clearer with named params.

Reply:
named parameters now added to finder 👍